app/helpers/contains-substring.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+
+/**
+ * containsSubstring helper
+ *
+ * @class containsSubstring
+ * @param {String} string The string being searched through
+ * @param {String} target The string being looked for
+ * @return {Boolean} Does the target string exist inside the specified string?
+ */
+export function containsSubstring(params/* , hash*/) {
+    const [string, target] = params;
+    let stringLower = string.toLowerCase();
+    let targetLower = target.toLowerCase();
+    if (stringLower.length == 0) {
```

Comment:
Probably not an issue here, but `===` is preferred over `==` when doing comparisons in javascript